content/en/docs/tasks/administer-cluster/migrating-from-dockershim/ migrating-telemetry-and-security-agents.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Migrating telemetry and security agents from dockershim
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+- jrcamp # for SignalFx (Splunk)
+weight: 70
+---
+
+<!-- overview -->
+
+Docker is being deprecated. From the [Dockershim Deprecation
+FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/) you might already
+know that most apps do not have a direct dependency on runtime hosting
+containers. However, there are still a lot of telemetry and security agents
+that has a dependency on docker to collect containers metadata, logs and
+metrics. This document aggregates information on how to detect these
+dependencies and links on how to migrate these agents to use generic tools or
+alternative runtimes.
+
+## Telemetry and security agents
+
+There are a few ways agents may be run on kubernetes cluster. Agents have a direct dependency on Docker when they run on nodes or as daemonsets. Some telemetry and security agents may rely on certain logs or metrics specific for Docker that would not be available any longer. This document does not cover these cases.
```

Comment:
nit: “Kubernetes” is a proper noun; we write object kinds like DaemonSet in UpperCamelCase.